usaspending_api/etl/es_etl_helpers.py

```diff
@@ -249,6 +252,44 @@ def streaming_post_to_es(client, chunk, index_name, job_id=None):
     return success, failed
 
 
+def put_alias(client, index, alias_name, award_type_codes):
+    alias_body = {'filter': {'terms': {"type": award_type_codes}}}
+    try:
+        client.indices.delete_alias(index, alias_name)
+    except TransportError as e:
+        print(e)
+    client.indices.put_alias(index, alias_name, body=alias_body)
+
+
+def reset_indices(client, new_index, old_index):
+    client.indices.refresh(new_index)
+    # add null values to contracts alias
+
+    indices_to_award_types['contracts'] += ('NULL',)
+
+    for award_type, award_type_codes in indices_to_award_types.items():
+        alias_name = '{}-{}'.format(settings.TRANSACTIONS_INDEX_ROOT, award_type)
+        printf({'msg': 'Putting alias "{}" with award codes {}'.format(alias_name, award_type_codes),
+                'job': '', 'f': 'ES Alias Put'})
+        put_alias(client, new_index, alias_name, award_type_codes)
+
+    old_index_patterns = old_index + '*'
+    old_indices = client.indices.get(old_index_patterns).keys()
+    for index in old_indices:
+        try:
+            client.indices.delete_alias(index, '_all')
+            client.indices.close(index)
+            printf({'msg': 'Closing index "{}"'.format(index), 'job': '', 'f': 'ES Index Close'})
+        except (TransportError, Exception) as e:
+            print(e)
```

Comment:
Clean up print and/or replace with SystemExit

Reply:
I created a `printf` function for logger-like output to the console. The elasticsearch library liked to add junk to the logger output and I didn't find a quick resolution to that.